src/Donations/PaymentMethods/PaymentFunctions.ts

```diff
@@ -215,32 +215,6 @@ export function createDonationData({
     };
   }
 
-  if (isGift) {
-    if (giftDetails.type === "invitation") {
-      donationData = {
-        ...donationData,
-        ...{
-          gift: {
-            type: "invitation",
-            recipientName: giftDetails.recipientName,
-            recipientEmail: giftDetails.recipientEmail,
-            message: giftDetails.message,
-          },
-        },
-      };
-    } else if (giftDetails.type === "direct") {
-      donationData = {
-        ...donationData,
-        ...{
-          gift: {
-            type: "direct",
-            recipient: giftDetails.recipient,
-          },
-        },
-      };
-    }
-  }
-
   if (projectDetails?.purpose === "planet-cash") {
```

Comment:
I wonder if your change deletes adding gift data to the API for all use cases and not just for the use case of topping up the planet-cash account? So maybe you disabled the full gift functionality with this PR rather the limiting it to planet-cash? Is this the wanted behavior?

Reply:
I wrote this comment on this line as you see we have some special handling for PlanetCash Top-up already implemented down here.

Reply:
The gift data need not be sent in any case. 
In cases where we don't need to top up planetcash, gift data was not being sent (which is the correct behavior)
i.e. if `isGift` is true, regardless of whether we're at `to=planetcash` or not, gift data should not be sent.
Though I may be wrong, @mariahosfeld could you check our comments?

Reply:
This has been resolved.